pkg/streamingpromql/testdata/ours/binary_operators.test

```diff
@@ -285,21 +280,21 @@ clear
 # whereby the histograms followed by a float are ignored and the lookback float value is used.
 # For example, in "another_mixed", at T=5m it has the lookback value of 10, and at T=5min the lookback value is _.
 load 5m
-  mixed_metric{job="test"}  10 1                                           2                                           3                                            {{schema:0 sum:6 count:3 buckets:[1 2 3]}} {{schema:0 sum:12 count:6 buckets:[2 4 6]}}
-  another_mixed{job="test"} 10 {{schema:0 sum:12 count:6 buckets:[2 4 6]}} {{schema:0 sum:12 count:6 buckets:[2 4 6]}} 4                                            5                                          {{schema:0 sum:12 count:6 buckets:[2 4 6]}}
-#                           @0 @5m                                         @10m                                        @15m                                         @20m                                       @25m
+  mixed_metric{job="test"}  10 1                                            2                                             3                                            {{schema:0 sum:6 count:3 buckets:[1 2 3]}}     {{schema:0 sum:12 count:6 buckets:[2 4 6]}}
+  another_mixed{job="test"} 10 {{schema:0 sum:12 count:6 buckets:[2 4 6]}}  {{schema:0 sum:12 count:6 buckets:[2 4 6]}}   4                                            5                                              {{schema:0 sum:12 count:6 buckets:[2 4 6]}}
+#                           @0 @5m                                          @10m                                          @15m                                         @20m                                           @25m
 
 eval range from 0 to 25m step 5m mixed_metric + another_mixed
-  {job="test"}              20 11                                          _                                           7                                            5                                          {{schema:0 sum:24 count:12 buckets:[4 8 12]}}
+  {job="test"}              20 _                                            _                                             7                                            _                                              {{schema:0 sum:24 count:12 buckets:[4 8 12]}}
 
 eval range from 0 to 25m step 5m mixed_metric - another_mixed
-  {job="test"}              0  -9                                          _                                           -1                                           -5                                         {{schema:0 sum:0 count:0}}
+  {job="test"}              0  _                                            _                                             -1                                           _                                              {{schema:0 sum:0 count:0}}
```

Comment:
These tests were deliberately checking the lookback values. We should change their step to 4m59s or load the values at 1m and use that as a step.

Reply:
Resolved in d457cc1